src/priiloader/source/titles.cpp

```diff
@@ -653,7 +644,7 @@ s32 LoadListTitles( void )
 			ClearScreen();
 			PrintFormat( 1, ((rmode->viWidth /2)-((strlen("Loading title..."))*13/2))>>1, 208, "Loading title...");
 
-			//lets start this bitch
+			//lets start this bitch // TODO: split out into LaunchTitle()
```

Comment:
no need to add todo, just split it :)

Reply:
i held off cos i was unsure about the error handling, but i wouldn't mind doing a bit more refactoring so will post another commit for this later, as long as the 3 in this pr are approved as a first step

Reply:
in that case, ill do the refactoring, and use exceptions like i do in the other code.